test/cli-commands/ganache/setup.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Do not copy this file but use the one in cli-commands. Otherwise on every change we would need to make the change here too